main.cpp

```diff
@@ -27,6 +27,10 @@ void ConfigureSurface() {
       .format = format,
       .width = kWidth,
       .height = kHeight};
+  // See TODO(dawn:2320) in dawn/native/vulkan/PhysicalDeviceVk.cpp
```

Comment:
That TODO doesn't exist anymore, maybe update your Dawn version?

Reply:
Ah, I see. I will close it then. I was looking at the dawn which is a submodule in this repository. It's tagged at chromium/6578 in https://github.com/beaufortfrancois/webgpu-cross-platform-app/blob/main/.gitmodules

Reply:
Ah that makes sense. @beaufortfrancois any issues with updating?

Reply:
I've had no issue updating to chromium/6613 on my linux machine. However, the demo works only when I enforce the usage of nvidia (instead of intel) gpu by hardcoding the `adapterRequestOptions.powerPreference=HighPerformance`. I've created an issue for intel gpu here https://github.com/beaufortfrancois/webgpu-cross-platform-app/issues/11

Reply:
I'll wait for Chrome 128 (chromium/6613) to be stable before updating this app if that's okay with you. I'll open a separate PR that updates .gitmodules dawn to chromium/6613 in the meantime.

Reply:
SGTM

Reply:
See https://github.com/beaufortfrancois/webgpu-cross-platform-app/pull/12